packages/core/src/config.ts

```diff
@@ -1,4 +1,4 @@
-import { UserConfig } from '@vocab/types';
+import type { UserConfig } from './types';
```

Comment:
Worth turning on the ESLint rule for this IMO (in a separate PR).

Reply:
https://github.com/seek-oss/eslint-config-seek/pull/96